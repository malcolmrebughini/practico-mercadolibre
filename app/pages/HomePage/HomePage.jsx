import React from 'react';
import { Helmet } from 'react-helmet';
import TopBar from '../../containers/TopBar';


function Home() {
  return (
    <div>
      <Helmet>
        <title>MercadoLibre</title>
        <meta
          name="description"
          content="La comunidad de compra y venta online más grande de América Latina."
        />
      </Helmet>
      <TopBar />
    </div>
  );
}


export default Home;
