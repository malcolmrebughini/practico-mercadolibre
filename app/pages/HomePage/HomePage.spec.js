/* eslint-env jasmine */
/* eslint react/jsx-filename-extension: 0 */

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomePage from './HomePage';


describe('HomePage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<HomePage />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
