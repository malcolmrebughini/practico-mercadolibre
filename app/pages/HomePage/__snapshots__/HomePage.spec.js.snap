// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render correctly 1`] = `
<div>
  <HelmetWrapper
    encodeSpecialCharacters={true}
  >
    <title>
      MercadoLibre
    </title>
    <meta
      content="La comunidad de compra y venta online más grande de América Latina."
      name="description"
    />
  </HelmetWrapper>
  <Connect(TopBar) />
</div>
`;
